src/browser_app_skeleton/spec/support/browser_client.cr.ecr

```diff
@@ -0,0 +1,12 @@
+class BrowserClient < Lucky::BaseHTTPClient
```

Comment:
Will this work correctly for people when they try to set params? I think since the content_type is set to text/html Lucky will not be able to parse params correctly. But I'm not sure. Same with the header stuff. I think it needs a session.

Unless `BrowserClient` works with params and auth as expected I think this should be removed and added later when we can make it work with sessions and params.

One way to handle params is to not set the content type and instead use application/json with json params. Otherwise we'll need to change the BaseHTTPClient (I think) to send form encoded params somehow

Reply:
Oh, you're saying when doing `BrowserClient.post()` because the params might be expecting the formenc type or whatever it is, this would clash... 🤔  good point. Ok, so for this PR then, we just rename `AppClient` to `ApiClient`, and in another PR add `BrowserClient` which can handle doing sessions and params and all that....

Reply:
Yeah exactly! 👍